src/style.css

```diff
@@ -0,0 +1,103 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.form {
+  margin: 20px 0;
+  position: relative;
+  display: block;
+  height: 68px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  bottom: 0;
+  right: 0;
+}
+
+.input {
+  font-size: 16px;
+  padding: 0 62px;
+  height: 68px;
+  width: 100%;
+  line-height: 10px;
+  border: none;
+  outline: none;
+  font-family: inherit;
+  font-weight: bold;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.input::placeholder {
+  font-weight: normal;
+  color: #3d4e61;
+}
+
+.small--form {
+  position: relative;
+  display: block;
+  height: 40px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.small--icon {
+  position: absolute;
+  top: 15px;
+  right: 0;
+  bottom: 0;
+  left: 13px;
+}
+
+.small--input {
+  font-size: 14px;
+  padding: 0 33px;
+  width: 100%;
+  height: 40px;
+  border: none;
+  outline: none;
+  font-family: inherit;
+  font-weight: bold;
+}
```

Comment:
you have a lot of similar styles with input
use DRY principle

Reply:
Thank`s for your review! Please check my new [DEMO LINK](https://maksym-yerokhin.github.io/layout_search-bar-airbnb/)